packages/terra-date-time-picker/CHANGELOG.md

```diff
@@ -3,6 +3,8 @@ ChangeLog
 
 Unreleased
 ----------
+### Fixed
```

Comment:
Could this be considered a breaking change?

Reply:
Ah, that is a great question! 

I am not sure. @benbcai @ryanthemanuel should this be a MVB?

Reply:
I think since this is a defect, while it's technically a change in behavior we are fixing the behavior to be "correct".  Let's leave it as a non MVB.